launch/exec_d_unix.go

```diff
@@ -0,0 +1,13 @@
+//+build linux darwin
+
+package launch
+
+import (
+	"os"
+	"os/exec"
+)
+
+func setHandle(cmd *exec.Cmd, pw *os.File) error {
+	cmd.ExtraFiles = []*os.File{pw}
```

Comment:
```suggestion
func setHandle(cmd *exec.Cmd, f *os.File) error {
	cmd.ExtraFiles = []*os.File{f}
```
This function doesn't need to know that the input file is a pipe writer.